conf/development.ini

```diff
@@ -9,8 +9,6 @@ pyramid.debug_notfound = false
 pyramid.debug_routematch = false
 pyramid.default_locale_name = en
 
-sqlalchemy.url = postgresql://postgres@localhost:5433/postgres
```

Comment:
Moved to tox.ini

Reply:
This does look like it might be worth doing in a separate PR (also given recent config changes).